adapters/adform/adform.go

```diff
@@ -513,9 +519,30 @@ func openRtbToAdformRequest(request *openrtb.BidRequest) (*adformRequest, []erro
 		consent:       consent,
 		digitrust:     digitrust,
 		currency:      requestCurrency,
+		eids:          eids,
 	}, errors
 }
 
+func encodeEids(eids *[]openrtb_ext.ExtUserEid) string {
```

Comment:
I recommend to keep this simple and pass in the slice directly. Then you don't need to dereference for then nil check or for loop.